com.unity.shadergraph/Editor/Importers/ShaderSubGraphImporter.cs

```diff
@@ -43,6 +44,7 @@ public override void OnImportAsset(AssetImportContext ctx)
             var subGraphPath = ctx.assetPath;
             var subGraphGuid = AssetDatabase.AssetPathToGUID(subGraphPath);
             graphAsset.assetGuid = subGraphGuid;
+            graphAsset.subGraphObjectIdOverride = "SubGraphImporter_subGraphData_luid_X";
```

Comment:
typo? should this read "guid" instead of "luid"?

Reply:
luid is just locally unique id.